core/src/main/scala/better/files/File.scala

```diff
@@ -370,8 +371,15 @@ class File private(val path: Path) {
     val algorithm = MessageDigest.getInstance(algorithmName)
     listRelativePaths.toSeq.sorted foreach { relativePath =>
       val file: File = path resolve relativePath
-      val bytes = if (file.isDirectory) relativePath.toString.getBytes else file.loadBytes
```

Comment:
maybe another solution would be to use `file.bytes` instead of `file.loadBytes` (former returns a Iterator[Byte]) and slurp it in chunks of size 1024?